### PR TITLE
Fixing Minimization Error In Dark Mode

### LINKED ISSUE #619: Fix Dark Mode Toggler UI
- Fix the toggler icon so that it stays together
Screen Shot 2020-09-01 at 5 50 09 PM
- Add toggler to the admin navbar

### PR DESCRIPTION
**Pull request by fmasood23**
Resolves #619

### CODE REVIEW

**Comment by fmasood23** [comment]
> the navbar's css got messed up
> Screen Shot 2020-09-02 at 12 02 56 AM

fixed this! this shouldn't happen anymore

**Comment by fmasood23** [comment]
> Screen Shot 2020-09-02 at 1 17 06 AM
> 
> The toggler is vertically aligned, could you vertically center it?

Let me know how it looks now!

**Review by jerrylee17 — APPROVED** [review]

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -117,12 +117,22 @@ export default function UserNavBar(props) {
             <img id='logo-image' src={logo} alt={'sce logo'} />
           </div>
         </NavbarBrand>
-        <div style={{ width: '0.7rem', height: '1.9rem' }} >{sunIcon()}</div>
-        <div style={{ width: '1.2rem' }} />
-        <CustomInput onChange={createCookie} className='darkToggle'
-          type='switch' id='exampleCustomSwitch' name='customSwitch'
-          checked={darkTheme} />
-        <div style={{ width: '1.1rem', height: '1.9rem' }} >{moonIcon()}</div>
+        <div align='left' style={{
```
Comment (IsitaB) [inline src/Components/Navbar/UserNavbar.js:120]: move styling to css file

**Review by IsitaB — CHANGES_REQUESTED** [review]
it works, but get rid of inline styling

**Review by aashrav — APPROVED** [review]
Looks good

**Review by IsitaB — APPROVED** [review]

**Comment by qsaml** [comment]
seems to work for me as well

**Review by IsitaB — APPROVED** [review]

**Pull request merged**